packages/widgetbook_generator/lib/src/code/widgetbook_category_instance.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:code_builder/code_builder.dart';
+
+import '../tree/tree_node.dart';
+import 'widgetbook_instance.dart';
+
+/// [InvokeExpression] for [WidgetbookCategory]
+class WidgetbookCategoryInstance extends WidgetbookInstance {
+  WidgetbookCategoryInstance({
+    required TreeNode<String> node,
+  }) : super(
+          type: 'WidgetbookCategory',
+          args: {
+            'name': literalString(_parseCategoryName(node.data)),
+            'children': literalList(
+              node.instances,
+            ),
+          },
+        );
+
+  static String _parseCategoryName(String name) {
+    final nameRegex = RegExp(r'^\[(.*?)\]$');
+    final match = nameRegex.firstMatch(name);
+
+    if (match == null) return name;
+
+    return match.group(1)!;
```

Comment:
Do you think using `node.data.substring(1, node.data.length - 1)` is more readable?
You can use an inline function if you agree with me.

Reply:
It probably is, yeah; I kept it a separate function, because I think it's more readable to keep the logic of stripping the brackets in a named section instead of inlining it with a ternary into the constructor. I hope that's alright!